spec/controllers/invite_requests_controller_spec.rb

```diff
@@ -41,19 +34,51 @@
       let(:invite_request) { create(:invite_request) }
 
       it "renders" do
-        get :show, params: { id: 0, email: invite_request.email }
+        get :show, params: { email: invite_request.email }
         expect(response).to render_template("show")
         expect(assigns(:invite_request)).to eq(invite_request)
       end
 
       it "renders for an ajax call" do
-        get :show, params: { id: 0, email: invite_request.email }, xhr: true
+        get :show, params: { email: invite_request.email }, xhr: true
         expect(response).to render_template("show")
         expect(assigns(:invite_request)).to eq(invite_request)
       end
     end
   end
 
+  describe "POST #resend" do
+    context "when the email doesn't match any invitations" do
+      it "redirects with an error" do
+        post :resend, params: { email: "test@example.org" }
+        it_redirects_to_with_error(status_invite_requests_path,
+                                   "Could not find an invitation associated with that email.")
+      end
+    end
+
+    context "when the invitation is too recent" do
+      let(:invitation) { create(:invitation) }
+
+      it "redirects with an error" do
+        post :resend, params: { email: invitation.invitee_email }
+        it_redirects_to_with_error(status_invite_requests_path,
+                                   "You cannot resend an invitation that was sent in the last 24 hours.")
+      end
+    end
+
+    context "when the email and time are valid" do
+      let!(:invitation) { create(:invitation) }
+
+      it "redirects with an error" do
+        travel_to (1 + ArchiveConfig.HOURS_BEFORE_RESEND_INVITATION).hours.since
```

Comment:
Nitpick:
```suggestion
        travel_to(1 + ArchiveConfig.HOURS_BEFORE_RESEND_INVITATION).hours.from_now
```
(mostly the spacing, IMO `from_now` is easier to understand but that's just me)